force-app/main/default/classes/AdyenConfigPageController.cls

```
@@ -51,6 +51,21 @@ public with sharing class AdyenConfigPageController {
         return paymentGatewayIds;
     }
 
+    @AuraEnabled
+    public static Map<String, String> getSetupPageUrls() {
+        Map<String, String> setupUrls = new Map<String, String>();
+        
+        String baseUrl = URL.getOrgDomainUrl().toExternalForm();
+        String checkoutNamedCredentialId = getNamedCredential(AdyenOMSConstants.CHECKOUT_NAMED_CREDENTIAL)?.Id;
```

Looks like this can be undefined? 
In that case since `getSetupPageUrls` is in a try/catch we can just throw an error instead of using the question mark?

---

Looks like this can be undefined?
In that case since getSetupPageUrls is in a try/catch we can just throw an error instead of using the question mark?